lib/opal/cli_runners/nodejs.rb

```diff
@@ -14,10 +15,13 @@ def self.call(data)
         argv = data[:argv].dup.to_a
         argv.unshift('--') if argv.any?
 
+        opts = Shellwords.shellwords(ENV['NODE_OPTS'] || "")
```

Comment:
nit: the linter is failing on this double quotes, can you please fix them?

Reply:
> nit: the linter is failing on this double quotes, can you please fix them?

Fixed